examples/ecs-fargate/django/app/infra/migrations/0001_install.py

```diff
@@ -14,12 +14,12 @@ class Migration(migrations.Migration):
         migrations.RunSQL(
             """
             SELECT * FROM iasql_install(
-                'aws_ecs_simplified'
+                'aws_ecs_simplified', 'aws_codebuild'
```

Comment:
why is this added if it is not in use?

Reply:
https://github.com/iasql/iasql-engine/pull/1682/files#diff-8818acbe0b7c48eb9a08c6d295a3de62882605533c5ebab4ed6b23bab3315f4aR149
It is used in `ecr_build` therefore we should install `aws_codebuild` to use it.

Reply:
oh, I see. Any reason to go with this pattern and not a hard dependency on the module?

Reply:
The module works without those modules installed (read/update/create/delete), so I thought we don't need to add the overhead to sync/apply by adding another module. Just the RPC depends on it so added the soft-dependency.